crates/polars-arrow/src/compute/cast/decimal_to.rs

```diff
@@ -12,15 +12,13 @@ fn decimal_to_decimal_impl<F: Fn(i128) -> Option<i128>>(
     to_precision: usize,
     to_scale: usize,
 ) -> PrimitiveArray<i128> {
-    let min_for_precision = 9_i128
-        .saturating_pow(1 + to_precision as u32)
-        .saturating_neg();
-    let max_for_precision = 9_i128.saturating_pow(1 + to_precision as u32);
```

Comment:
It's pretty bizarre to me that someone wrote this logic. It makes absolutely no sense to me. Am I missing something?